src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2379 @@ const ActorInit En_Ossan_InitVars = {
+                                     { 0x0010, 0x0050, 0x004C, 0xFFFD },
+                                     { 0x0012, 0xFFCE, 0x0034, 0xFFEC },
+                                     { 0x0005, 0xFFCE, 0x004C, 0xFFEC },
+                                     { 0x0010, 0xFFB0, 0x0034, 0xFFFD },
+                                     { 0x0010, 0xFFB0, 0x004C, 0xFFFD },
+                                 },
+                                 {
+                                     { 0x000F, 0x0032, 0x0034, 0xFFEC },
+                                     { 0x002C, 0x0032, 0x004C, 0xFFEC },
+                                     { 0x0010, 0x0050, 0x0034, 0xFFFD },
+                                     { 0x0001, 0x0050, 0x004C, 0xFFFD },
+                                     { 0x0000, 0xFFCE, 0x0034, 0xFFEC },
+                                     { 0x0002, 0xFFCE, 0x004C, 0xFFEC },
+                                     { 0x0007, 0xFFB0, 0x0034, 0xFFFD },
+                                     { 0x0031, 0xFFB0, 0x004C, 0xFFFD },
+                                 },
+                                 {
+                                     { 0x0003, 0x0032, 0x0034, 0xFFEC },
+                                     { 0x0006, 0x0032, 0x004C, 0xFFEC },
+                                     { 0x002D, 0x0050, 0x0034, 0xFFFD },
+                                     { 0x002E, 0x0050, 0x004C, 0xFFFD },
+                                     { 0x000E, 0xFFCE, 0x0034, 0xFFEC },
+                                     { 0x0010, 0xFFCE, 0x004C, 0xFFEC },
+                                     { 0x0030, 0xFFB0, 0x0034, 0xFFFD },
+                                     { 0x0010, 0xFFB0, 0x004C, 0xFFFD },
+                                 },
+                                 {
+                                     { 0x0013, 0x0032, 0x0034, 0xFFEC },
+                                     { 0x0013, 0x0032, 0x004C, 0xFFEC },
+                                     { 0x0013, 0x0050, 0x0034, 0xFFFD },
+                                     { 0x0013, 0x0050, 0x004C, 0xFFFD },
+                                     { 0x0014, 0xFFCE, 0x0034, 0xFFEC },
+                                     { 0x0014, 0xFFCE, 0x004C, 0xFFEC },
+                                     { 0x0014, 0xFFB0, 0x0034, 0xFFFD },
+                                     { 0x0014, 0xFFB0, 0x004C, 0xFFFD },
+                                 },
+                                 { { 0x0025, 0x0032, 0x0034, 0xFFEC },
+                                   { 0x0023, 0x0032, 0x004C, 0xFFEC },
+                                   { 0x0022, 0x0050, 0x0034, 0xFFFD },
+                                   { 0x0024, 0x0050, 0x004C, 0xFFFD },
+                                   { 0x0020, 0xFFCE, 0x0034, 0xFFEC },
+                                   { 0x001E, 0xFFCE, 0x004C, 0xFFEC },
+                                   { 0x0021, 0xFFB0, 0x0034, 0xFFFD },
+                                   { 0x001F, 0xFFB0, 0x004C, 0xFFFD } } };
```

Comment:
to stop the formatter from doing this dumb formatting, but a trailing comma on the block of numbers